src/vendor/github.com/xelabs/go-mysqlstack/driver/server.go

```diff
@@ -212,6 +213,7 @@ func (l *Listener) handle(conn net.Conn, ID uint32, serverVersion string) {
 			}
 		case sqldb.COM_QUERY:
 			query := l.parserComQuery(data)
+			session.UpdateTime(time.Now())
```

Comment:
session.UpdateTime(time.Now())
query := l.parserComQuery(data)

is better?

Reply:
ok